src/test/java/org/pkcs11/jacknji11/CryptokiTest.java

```diff
@@ -21,10 +21,14 @@
 
 package org.pkcs11.jacknji11;
 
-import java.nio.charset.StandardCharsets;
 import java.util.Arrays;
+import java.util.function.Predicate;
 
 import junit.framework.TestCase;
+import org.pkcs11.jacknji11.GetAttributeLengthStrategy.Default;
+
+import static org.pkcs11.jacknji11.CKA.MODULUS;
```

Comment:
Why the static import?

Reply:
Thats just Idea managing imports automatically. Wanna qualify it instead?

Reply:
Done.

Reply:
I think qualifying it makes it more consistent. I.e. CKA.PUBLIC_EXPONENT vs MODULUS_BITS.

Reply:
yes, please remove the static import for consistency

Reply:
Done in whole PR except for for CK and CKA classes.